### PR TITLE
Start on some header documentation for masse

### PR DESCRIPTION
**Pull request by BasThomas**

### CODE REVIEW

File: masse.swift [hunk 1]
```diff
@@ -40,8 +51,16 @@ extension String {
...
+
+    /// Gets a String between two indices.
+    ///
+    /// - Precondition: The `beginPosition` must be smaller than the `endPosition`.
+    ///
+    /// - Parameter beginPosition: The beginning position's index.
+    /// - Parameter endPosition: The end position's index.
+    /// - Returns: The String between `beginPosition` and `endPosition`.
     func between(beginPosition: Int, endPosition: Int) -> String {
+        precondition(beginPosition < endPosition, "The beginning position must be smaller than the end position.")
```
Comment (BasThomas) [inline masse.swift]: We should add some tests, this is super easy to crash right now 😛

**Comment by BasThomas** [comment]
It seems like single Swift files do not support syntax highlighting or header documentation checks, ugh.

**Comment by terhechte** [comment]
The trick is to put them into a playground. I have an extended playground including tests. Will share tomirrow

**Comment by BasThomas** [comment]
I did exactly that :)

**Comment by BasThomas** [comment]
Closing this to move these changes to the `masse` repository.

**Pull request closed without merging**